pxc/jenkins/qa_pxc_57_80_test-pipeline.groovy

```diff
@@ -0,0 +1,163 @@
+pipeline_timeout = 10
+
+pipeline {
+    parameters {
+        string(
+            defaultValue: '5.7.38-31.59',
+            description: 'PXC lower version tarball to download for testing',
+            name: 'PXC_LOWER_VERSION_TAR',
+            trim: true)
+        string(
+            defaultValue: '8.0.28-19.1',
+            description: 'PXC Upper version tarball to download for testing',
+            name: 'PXC_UPPER_VERSION_TAR')
+        string(
+            defaultValue: '5.7.38-rel41-59.1',
+            description: 'PXC-5.7 package version',
+            name: 'PXC57_PKG_VERSION',
+            trim: true)
+        string(
+            defaultValue: 'https://github.com/Percona-QA/percona-qa',
+            description: 'URL to Percona-QA repository',
+            name: 'PERCONA_QA_REPO',
+            trim: true)
+        string(
+            defaultValue: 'master',
+            description: 'Tag/Branch for Percona-QA repository',
+            name: 'BRANCH',
+            trim: true)
+        choice(
+            choices: [
+                'min-centos-7-x64',
+                'min-ol-8-x64',
+                'min-bionic-x64',
+                'min-focal-x64',
+                'min-buster-x64',
+                'min-bullseye-x64'
+            ],
+            description: 'Node to run tests',
+            name: 'node_to_test'
+        )
+    }
+    agent {
+        label params.node_to_test
+    }
+    options {
+        skipDefaultCheckout()
+        skipStagesAfterUnstable()
+        timeout(time: 6, unit: 'DAYS')
+        buildDiscarder(logRotator(numToKeepStr: '200', artifactNumToKeepStr: '200'))
+    }
+stages {
+        stage("Build Distribution") {
+            agent { label params.node_to_test }
+    stages {
+        stage('Prepare') {
+            steps {
+                sh 'echo Downloading PXC LOWER VERSION tarball: \$(date -u)'
+                sh '''
+                    echo "Installing dependencies..."
+                    if [ -f /usr/bin/yum ]; then 
+                    sudo yum -y update
+                    sudo yum install -y git wget tar socat
+                    else
+                    lsb_release -a
+                    sudo apt install -y git wget socat curl gnupg2 numactl
```

Comment:
Why aee we installing gnupg2 ?

Reply:
In Debian distributions errors were occurring because of the absence of GnuPG pkg , thats why we are installing gnupg2 here

Reply:
Ok